scalafix-tests/output/src/main/scala/test/RemoveUnusedTerms.scala

```diff
@@ -0,0 +1,13 @@
+package test
+
+object RemoveUnusedTerms {
+
+  def foo {
+    
+    println(5)
+    println(0)
+    println(1)
+  }
+
+  val unused = 0
```

Comment:
Why does this stay?

Reply:
As far as I could see, this wasn't flagged by the compiler as unused. But I have to admit this looks a bit weird. // cc @olafurpg ?

Reply:
This is a globally accessible val, it could be used by another project.